common/endpoint-shared/src/main/java/org/kaaproject/kaa/common/channels/protocols/kaatcp/messages/MessageFactory.java

```diff
@@ -81,154 +72,109 @@ public MessageFactory() {
     @Override
     public void onMqttFrame(MqttFrame frame) {
         switch (frame.getMessageType()) {
-        case CONNACK:
-            if (connAckListener != null) {
-                connAckListener.onMessage((ConnAck) frame);
-            }
-            break;
-        case CONNECT:
-            if (connectListener != null) {
-                connectListener.onMessage((Connect) frame);
-            }
-            break;
-        case DISCONNECT:
-            if (disconnectListener != null) {
-                disconnectListener.onMessage((Disconnect) frame);
-            }
-            break;
-        case KAASYNC:
-
-            onKaaSyncMessage((KaaSync) frame);
-
-            break;
-        case PINGREQ:
-            if (pingRequestListener != null) {
-                pingRequestListener.onMessage((PingRequest) frame);
-            }
-            break;
-        case PINGRESP:
-            if (pingResponseListener != null) {
-                pingResponseListener.onMessage((PingResponse) frame);
-            }
-            break;
-        default:
-            break;
+            case CONNACK:
```

Comment:
Please try to avoid such "formatting" changes. This increase complexity of your PR and makes us spend more time on review.